### ISSUE TITLE
[iOS] no other people video and audio, but I can see me on my ios device 

### ISSUE THREAD

**Issue by Liuchunqi3240** [jitsi/jitsi-meet#8395]
```
    JitsiMeetView *jitsiView = (JitsiMeetView*)self.view;
    jitsiView.delegate = self;
    
    // Join the room.
    JitsiMeetConferenceOptions *options
        = [JitsiMeetConferenceOptions fromBuilder:^(JitsiMeetConferenceOptionsBuilder *builder) {
            builder.serverURL = [NSURL URLWithString:@"https://meet.jit.si"];
            builder.room = self.room;
        }];
    [jitsiView join:options];

```
sample is work 
this repo ios code , not work

**Comment by Liuchunqi3240** [comment]
I found error on chrom , channel problem .
that's reasion is sdp ?
I use the docker-jitsi-meet-stable-5142,
what can I choose  version or commitID

**Comment by Liuchunqi3240** [comment]
![IMG_D49734773BDE-1](https://user-images.githubusercontent.com/13010614/104537161-6106a880-5654-11eb-8ae9-1d829bc3c5bb.jpeg)
this image is my device display

**Comment by Liuchunqi3240** [comment]
the mac user  is other people

**Comment by Liuchunqi3240** [comment]
![image](https://user-images.githubusercontent.com/13010614/104543606-c6609680-5660-11eb-9f42-9a2d839f5311.png)
the web page display this icon.

**Comment by Liuchunqi3240** [comment]
HELP....

**Comment by Liuchunqi3240** [comment]
ok , I find a available version， tag is `ios-sdk-2.11.0`
